docs/source/configuring-slack-for-spidermon.rst

```diff
@@ -0,0 +1,31 @@
+.. _configuring-slack-bot:
+
+====================================
+Configure Slack bot for Spidermon
+====================================
```

Comment:
It’s important that the equal signs here match the length of the title. I believe the documentation build fails otherwise.

Reply:
Counted and trimmed.

Reply:
It’s actually 1 character too long, however I checked and the documentation build does not seem to care. Maybe when there are equal signs both above and below it’s not an issue.